src/os/bluestore/StupidAllocator.cc

```diff
@@ -160,9 +160,9 @@ int64_t StupidAllocator::allocate_int(
     skew = alloc_unit - skew;
   *offset = p.get_start() + skew;
   *length = MIN(MAX(alloc_unit, want_size), P2ALIGN((p.get_len() - skew), alloc_unit));
-  if (cct->_conf->bluestore_debug_small_allocations) {
+  if (cct->_conf->get_val<int64_t>("bluestore_debug_small_allocations")) {
```

Comment:
@liewegas shall we cache this setting in `StupidAllocator`? i feel that this is a critical path. and `get_val<>()` acquires and releases a lock when it's called.

Reply:
yeah, this is critical path and needs a local variable and config observer